outputs.tf

```diff
@@ -1,5 +1,5 @@
-output "ansible-control_access_ip_v4" {
-  value = openstack_compute_instance_v2.ansible-control.access_ip_v4
+output "ansible_control_access_ip_v4" {
```

Comment:
nit: name is inconsistent with the others

Reply:
Do we actually need these outputs?

Reply:
This could possibly be cleaned up as they all get written to file and don't need to be outputted to terminal. Possibly keep the ansible IP so you know what to connect to?